FlowCrypt/Functionality/Services/Remote Pub Key Services/AttesterApi.swift

```diff
@@ -7,8 +7,8 @@ import FlowCryptCommon
 
 protocol AttesterApiType {
     func lookup(email: String) async throws -> [KeyDetails]
-    func update(email: String, pubkey: String, token: String?) async throws -> String
-    func replace(email: String, pubkey: String) async throws -> String
+    func replace(email: String, pubkey: String, idToken: String?) async throws -> String
+    func update(email: String, pubkey: String) async throws -> String
```

Comment:
Replace means change one pubkey to another (needs Id token). Update means upload an updated version of the same public key (same primary fingerprint) - no id token needed.